src/common/daft-config/src/lib.rs

```diff
@@ -41,6 +44,9 @@ impl Default for DaftExecutionConfig {
             merge_scan_tasks_max_size_bytes: 512 * 1024 * 1024, // 512MB
             broadcast_join_size_bytes_threshold: 10 * 1024 * 1024, // 10 MiB
             sample_size_for_sort: 20,
+            split_row_groups_max_files: 10,
+            split_row_groups_threshold_bytes: 24 * 1024 * 1024, // 24MB
```

Comment:
these thresholds are fairly low, We typically target each rest request to be about 16mb. Also Row groups are typically around 128mb. So we should probably have the `split_row_groups_min_size_bytes` be something like 64mb and the `split_row_groups_threshold_bytes` be around 128mb